src/identity/interaction/email-password/handler/LoginHandler.ts

```diff
@@ -22,34 +21,26 @@ export class LoginHandler extends InteractionHandler {
 
   public async handle({ operation }: InteractionHandlerInput): Promise<InteractionCompleteResult> {
     const { email, password, remember } = await this.parseInput(operation);
-    try {
-      // Try to log in, will error if email/password combination is invalid
-      const webId = await this.accountStore.authenticate(email, password);
-      this.logger.debug(`Logging in user ${email}`);
-      return {
-        type: 'complete',
-        details: { webId, shouldRemember: Boolean(remember) },
-      };
-    } catch (err: unknown) {
-      throwIdpInteractionError(err, { email });
-    }
+    // Try to log in, will error if email/password combination is invalid
+    const webId = await this.accountStore.authenticate(email, password);
+    this.logger.debug(`Logging in user ${email}`);
+    return {
+      type: 'complete',
+      details: { webId, shouldRemember: Boolean(remember) },
```

Comment:
Do we still need to cast?

Reply:
We don't know the typing of the `remember` parameter, and in practice it is still going to be the `'on'` string sometimes since form data to JSON conversion doesn't change that.

Reply:
But it's a return value from `parseInput`?

Reply:
Ah yea, you're right. I missed that.